### DISCUSSION TITLE
Start / build and SOA : Add an option to ingore webpackage of strapi-admin for specific env

### DISCUSSION THREAD

**Discussion by Erim32** [strapi/strapi#5456] [Legacy Topics]
Hi, thank you for your work,
this feature request is probably easy to implement and can be really useful.
The idea is to add an option in config file, in admin section to disable the web packaging of strapi-admin in specific running environment.

This option can bring 2 advantages : 
-  Improve security : prevent access to the administration front then access to data if the admin folder is not included in the web project in production (for example).
- Ability to create small micro services projects (basically only database and controllers can be included in a build without any front except endpoints ). 

[https://blogs.bmc.com/wp-content/uploads/2018/10/microservices-vs-monolithic-1024x544.png](monolithic architecture vs micro-services) 
Thank you

Path — ./config/environment/**/server.json
`json
{
  "host": "localhost",
  "port": 1337,
  "autoReload": {
    "enabled": true
  },
  "cron": {
    "enabled": false
  },
  "admin": {
    "enabled": false,
    "path": "/admin"
  }
}
`

**Comment by derrickmehaffy** [comment]
@Aurelsicoko @JabStrapi Is this something we want to stick on the product board?
